test/test_dht_storage.cpp

```diff
@@ -71,6 +72,12 @@ namespace
 		from_hex(s, 40, (char *) &ret[0]);
 		return ret;
 	}
+
+	dht_storage_interface* dht_custom_storage_constructor(sha1_hash const& id
+		, dht_settings const& settings)
+	{
+		return dht_default_storage_constructor(id, settings);
```

Comment:
ah, I see, this just forwards. I suppose you could set a global variable or something here, to communicate back with the test